frontend/config/application.coffee

```diff
@@ -8,8 +8,8 @@
 
 module.exports = require(process.env['LINEMAN_MAIN']).config.extend('application', {
   removeTasks:
-    common: [ "webfonts:dev", "images:dev"]
-    dist: ["images:dist", "webfonts:dist", "pages:dist"]
+    common: ['copy:dev', 'webfonts:dev', 'images:dev', 'concat_sourcemap', 'pages:dev']
```

Comment:
`concat_sourcemap` is removed from common here to remove `concat_sourcemap:css` (as without an argument it runs all the `concat_sourcemap` tasks creating an empty css file. It's added back in for js and spec [in appendTasks](https://github.com/kevindew/exercism.io/blob/81a9ce8af586cd0ecb00325821be385dc797a544/frontend/config/application.coffee#L30-L31)